modules/delegates.js

```diff
@@ -52,6 +52,8 @@ function Delegates (cb, scope) {
 		config: {
 			forging: {
 				secret: scope.config.forging.secret,
+				force: scope.config.forging.force,
+				defaultKey: scope.config.forging.defaultKey,
```

Comment:
secrets will be stored in an encrypted format in `config.json` file. Like this: 
`
  {
    "encryptedSecret": <passphrase>
    "publicKey": <publicKey>
  }
`
This `defaultKey` is used to decrypt those secrets on application startup to load delegates for testnet/devnet.